pkg/asset/manifests/clusterapi/cluster.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/Azure/go-autorest/autorest/to"
```

Comment:
I will change this to a different package

Reply:
Can this one be used?
"k8s.io/utils/pointer"